src/HealthGPS.Console/jsonparser.h

```diff
@@ -81,4 +82,25 @@ void from_json(const json &j, PolicyScenarioInfo &p);
 // Output information
 void to_json(json &j, const OutputInfo &p);
 void from_json(const json &j, OutputInfo &p);
+
 } // namespace host::poco
+
+namespace std {
+
+// Optional parameters
+template <typename T> void to_json(nlohmann::json &j, const std::optional<T> &p) {
```

Comment:
Good templating!

I'm a little surprised the library doesn't include this by default...